manifests/client.pp

```diff
@@ -1,8 +1,9 @@
 class amanda::client (
-  $remote_user      = undef,
-  $server           = "backup.${facts['networking']['domain']}",
-  $xinetd           = true,
-  $export_host_keys = false,
+  String           $ensure           = 'present',
```

Comment:
Maybe we can use a type `Enum['present', 'absent']`

Reply:
Okay, I changed the type to an Enum.